xml/cl.xml

```diff
@@ -5831,6 +5842,16 @@ server's OpenCL/api-docs repository.
             </require>
         </extension>
         <extension name="cl_arm_scheduling_controls" supported="opencl">
+            <require comment="Types">
```

Comment:
I think `CL_DEVICE_SUPPORTED_REGISTER_ALLOCATIONS_ARM` should be included somewhere here, perhaps in the `cl_device_info` block?

We can merge these changes as-is since this won't break anything right now, but it will be needed when we start generating extension headers (hopefully soon!).

Reply:
It looks like we're also missing `CL_QUEUE_DEFERRED_FLUSH_ARM`.

Reply:
Good catch! I've now added them both.

Another advantage of generation is that it will provide us with stronger guarantees on the correctness/completeness of the XML.

Reply:
Yeah, I've been (manually) generating the headers each time to ensure the XML and header changes are consistent.  It's good motivation to get the header generation across the finish line so I can stop doing this.  😃